lib/filter/filter-re.c

```diff
@@ -179,8 +179,10 @@ filter_match_set_template_ref(FilterExprNode *s, LogTemplate *template)
 }
 
 static gboolean
-filter_match_eval_against_program_pid_msg(FilterMatch *self, LogMessage **msgs, gint num_msg)
+filter_match_eval_against_program_pid_msg(FilterExprNode *s, LogMessage **msgs, gint num_msg)
```

Comment:
For other reviewers: the motivation is these functions will be used as virtual fuction